flask_rest_api/blueprint.py

```diff
@@ -61,6 +61,7 @@ class Blueprint(
     def __init__(self, *args, **kwargs):
 
         self.description = kwargs.pop('description', '')
+        self.docstring_sep = kwargs.pop('docstring_sep', '---')
```

Comment:
+ Is this where you were thinking of adding the arg?
+ Where are the docs for the Blueprint so that I can add it there?

Reply:
I was thinking of making it a class attribute the user could override in a subclass, like `HTTP_METHODS`.

```py
class Blueprint:

    DOCSTRING_SEP = '---'
```
I'd document this in docs/openapi.rst.

Reply:
Do you still want me to move it to a class attribute rather than (or in addition to) an initialization arg?

Reply:
Class attribute only would be consistent with the rest of the lib.

The separator will most probably be something the user will use consistently in all his `Blueprint`s so subclassing and using a class attribute seems nice to me.

`description` is obviously different because it differs in all instances.

Reply:
Sounds good. I added it as a class attribute.

I also left the initialization arg so that users can do any of these:

```python
# No need to subclass
blp0 = Blueprint('test', __name__, url_prefix='test', docstring_sep="~~~")

# But you can if you want to
class MyBlueprint(Blueprint):
    DOCSTRING_SEP = "~~~"

blp1 = MyBlueprint('test', __name__, url_prefix='test')

# and you can still override your subclass on a per-instance bases if desired.
blp2 = MyBlueprint('test', __name__, url_prefix='test', docstring_sep="^^^")
```

If that's not to your liking please let me know and I'll remove the init arg completely.